### DISCUSSION TITLE
After a few hours, I made it!

### DISCUSSION THREAD

**Discussion by Woody123123** [utmapp/UTM#6395] [Show and tell]
![截圖 2024-05-27 下午11 17 38](https://github.com/utmapp/UTM/assets/110503402/3a50f19e-2dc4-440b-ab52-5b6b9e9073f1)
macOS X Catalina on M1 mac!

**Comment by Woody123123** [comment]
.utm link here https://drive.google.com/file/d/1HPmJ2ZtYnej1dRtzez8pUkWhV6Oy8LKD/view?usp=sharing

**Reply by powermac84** [reply]
EFI-LEGACY.qcow2 is missing ;(

**Comment by powermac84** [comment]
And what is the password to the user "Woody"?

**Reply by Woody123123** [reply]
2021

**Reply by Woody123123** [reply]
https://github.com/adespoton/utmconfigs/blob/main/macOS%2010.14%20(config%20only).utm.zip
